### PR TITLE
chore: refactor code to meet pylint style guidelines. WIP

### PR DESCRIPTION
**Pull request by chtnnh**
#34 

Sorry, I may have jumped the gun on this @rafaeelaudibert but it was really bugging me.

Also, might need your help with setting up the GitHub CI to check for pylint errors

### CODE REVIEW

File: encryptor/__init__.py [hunk 1]
```diff
@@ -48,113 +55,86 @@ def create_app(test_config=None):
         pass
 
     # ROUTES
+    # pylint: disable=unused-variable
```
Comment (rafaeelaudibert) [inline encryptor/__init__.py:58]: Which variable are we ignoring?. This doesn't seem to be needed

File: encryptor/__init__.py [hunk 2]
```diff
@@ -48,113 +55,86 @@ def create_app(test_config=None):
...
-        })
+        return jsonify({"status": 200, "content": Blowfish.decrypt(text, mode)})
 
-          
     @app.route("/api/rsa/encrypt/<text>")
     def rsa_encrypt(text):
-        n = request.args.get("n", Rsa.DEFAULT_N)
-        e = request.args.get("e", Rsa.DEFAULT_E)
+        input_n = request.args.get("n", Rsa.DEFAULT_N)
+        input_e = request.args.get("e", Rsa.DEFAULT_E)
```
Comment (rafaeelaudibert) [inline encryptor/__init__.py:115]: 👍🏻

File: encryptor/encryptors/blowfish.py [hunk 3]
```diff
@@ -1,65 +1,96 @@
...
-class BlowfishException(werkzeug.exceptions.BadRequest):
+class BlowfishException(BadRequest):
+
+    """
+    Class defining custom Blowfish Exception
+    """
+
     def __init__(self, message):
-        self.message = message
+        BadRequest.message = message
```
Comment (rafaeelaudibert) [inline encryptor/encryptors/blowfish.py:24]: Why do we need this change? We could do it differently, though, calling `super.__init__(message)`

File: encryptor/encryptors/vigenere.py [hunk 4]
```diff
@@ -1,48 +1,90 @@
...
+
     # use ASCII by default
     ALPHABET = {i: chr(i) for i in range(128)}
-    KEY = 'abra'
+    KEY = "abra"
 
     @staticmethod
     def encrypt(text: str, key: str = KEY, tabula_recta=None):
+        """
+        Viegenere - Encrypt
```
Comment (rafaeelaudibert) [inline encryptor/encryptors/vigenere.py:27]:
```suggestion
        Vigenere - Encrypt
```

File: encryptor/encryptors/vigenere.py [hunk 5]
```diff
@@ -1,48 +1,90 @@
...
+
+        Call helper functions to encrypt input string
+        """
+
         if tabula_recta is None:
             tabula_recta = Vigenere.ALPHABET
         if key.isalpha():
```
Comment (rafaeelaudibert) [inline encryptor/encryptors/vigenere.py:34]: We could use some spacing here
```suggestion
            tabula_recta = Vigenere.ALPHABET
        
        if key.isalpha():
```

File: encryptor/encryptors/vigenere.py [hunk 6]
```diff
@@ -1,48 +1,90 @@
...
+                for value in compare.values()
+            ]
+            return "".join(Vigenere._decode(encrypt_message))
+
+        raise NotAllowedValue
 
     @staticmethod
     def decrypt(text: str, key: str = KEY, tabula_recta=None):
+        """
+        Viegenere - Decrypt
```
Comment (rafaeelaudibert) [inline encryptor/encryptors/vigenere.py:49]:
```suggestion
        Vigenere - Decrypt
```

**Review by rafaeelaudibert — CHANGES_REQUESTED** [review]
This looks incredible @chtnnh! Thank you so much for contributing. I've left a review below, with some questions and suggested changes.

I think we could leave the CI for a next PR, just so that this PR doesn't get even bigger. What are your thoughts?

File: encryptor/__init__.py (hunk 1 above)
Comment (chtnnh) [reply encryptor/__init__.py:58]: This is for the routes declared as functions. Pylint classifies them as unused-variables

File: encryptor/encryptors/blowfish.py (hunk 3 above)
Comment (chtnnh) [reply encryptor/encryptors/blowfish.py:24]: Linter threw an error I didn't fully understand. But this made it go away 😆 

LMK if this is unacceptable

**Pull request closed without merging**